setup.cfg

```diff
@@ -4,6 +4,7 @@ version = attr: pytest_aiohttp.__version__
 url = https://github.com/aio-libs/pytest-aiohttp
 project_urls =
   GitHub = https://github.com/aio-libs/pytest-aiohttp
+  Changes = https://github.com/aio-libs/pytest-aiohttp/blob/master/CHANGES.rst
```

Comment:
Could you call this `Changelog` and put first in the list to keep it sorted?

Reply:
The doc's [title is CHANGES](https://github.com/aio-libs/pytest-aiohttp/blob/master/CHANGES.rst#changes), I think it's better to keep in sync with that.

I believe the links over at PyPI will be displayed in the order they're specified here, and to me listing Changes after GitHub sounds like the right thing to do.

Reply:
I think yes, PyPI displays things sorted.
FWIW this project will probably follow aiohttp's lead exposing the document as `Changelog` on the web: https://docs.aiohttp.org/en/stable/changes.html.

Reply:
Let's follow aiohttp